localization/english/culture/ctp_titles_l_english.yml

```diff
@@ -1146,33 +1146,34 @@ empire_feudalclan_serbian:0 "Carstvo"
 
 #vlach
 
-baron_feudalclan_male_vlach:0 "Baron"
-baron_republic_male_vlach:0 "Primar" 
-baron_feudalclan_female_vlach:0 "Baronesă" 
-baron_republic_female_vlach:0 "Primăreasă"
-barony_feudalclan_vlach:0 "Baronie"
+baron_feudalclan_male_vlach:0 "Castelan"
+baron_republic_male_vlach:0 "Jude" # Used in Wallachia. In Moldova it was "Șoltuz" (romanianised variant of the transylvanian saxon "schulteiss" (mayor))
+baron_feudalclan_female_vlach:0 "Castelană"
+baron_republic_female_vlach:0 "Jude" # I'm not aware of any feminine variant but the original one is pretty gender-neutral
+barony_feudalclan_vlach:0 "Cetate"
```

Comment:
This one is also kinda suspect - ain't it "civitas", which is whatever but not feudal?

Reply:
Cetate is a fortress / stronghold / castle